util/reggen/gen_cheader.py

```diff
@@ -188,6 +205,43 @@ def gen_cdefines_module_params(outstr, module_data, module_name,
     genout(outstr, '\n')
 
 
+def gen_multireg_field_defines(outstr, regname, field, subreg_num, regwidth,
+                               existing_defines):
+    field_width = field['bitinfo'][1]
+    fields_per_reg = int(regwidth / field_width)
```

Comment:
This always rounds down, so you can just use the following for integer division:
```suggestion
    fields_per_reg = regwidth // field_width
```

Reply:
Oh cool, thanks.

Reply:
Done.